### ISSUE TITLE
npm-test-integ cannot run in the pipeline

### ISSUE THREAD

**Issue by eladb** [aws/aws-cdk#6328]
The new CLI test `npm-test-integ`, which was introduced as part of #5856 cannot be executed as part of our CI pipeline because it assumes we have a development environment (`cdk-integ` for example).

We removed this test in https://github.com/aws/aws-cdk/pull/6327 to unblock our builds, but we need to figure what to do about it.

### Error Log

```
============================================================================================
--
972 | test-npm-integ.sh
973 | ============================================================================================
974 |  
975 | > aws-cdk@1.24.0 integ /codebuild/output/src676395972/src/package
976 | > jest --testMatch '**/?(*.)+(integ-test).ts'
977 |  
978 | sh: jest: command not found
979 | npm ERR! code ELIFECYCLE
980 | npm ERR! syscall spawn
981 | npm ERR! file sh
982 | npm ERR! errno ENOENT
983 | npm ERR! aws-cdk@1.24.0 integ: `jest --testMatch '**/?(*.)+(integ-test).ts'`
984 | npm ERR! spawn ENOENT
985 | npm ERR!
986 | npm ERR! Failed at the aws-cdk@1.24.0 integ script.
987 | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
988 | npm WARN Local package.json exists, but node_modules missing, did you mean to install?
989 |  
990 | npm ERR! A complete log of this run can be found in:
991 | npm ERR!     /root/.npm/_logs/2020-02-18T05_01_24_571Z-debug.log
```

--- 

This is :bug: Bug Report

**Comment by rix0rrr** [comment]
Thinking on this some more... an "npm install" of jest and `@aws-cdk/core` etc should be enough. It will be read proxied through verdaccio. The `.ts` will have been compiled to `.js` as part fo the regular package build.

I'm leaning towards a separate `package.json` for integ test dependencies, but we'll have to hide that from `lerna`. Or shouldn't we?

**Comment by eladb** [comment]
What about `cdk-integ`, which is not published publicly?

**Comment by rix0rrr** [comment]
What about it?

**Comment by eladb** [comment]
Doesn't the integration test use it?

**Comment by rix0rrr** [comment]
No. We have 2 completely different things we both call "integration tests".

We should rename one of them. "template tests" or something.

**Comment by skinny85** [comment]
This one was fixed in #7041.
